blocks/columns/columns.css

```diff
@@ -58,6 +58,104 @@ div.column-collections-parent {
   margin-bottom: 5px;
 }
 
+div.columns-wrapper.old-magazine-header-wrapper {
+  max-width: 80vw;
+}
+
+.columns.old-magazine-header.block .header-content{
+  animation: slide-in-from-right 1s ease-out forwards;
+  transform:translateX(-300%);
+  opacity: 0;
+}
+
+
+@keyframes slide-in-from-right {
```

Comment:
looks great. We don't need the animation, we agreed with the customer that it's a better experience without it

Reply:
sure I will take it out in a min